src/web/Fig.Web/Components/Assistant/FigAssistantHost.razor.css

```
@@ -0,0 +1,337 @@
+    white-space: pre-wrap;
+    background: #3f5f79;
+}
+
+.assistant-message {
+    align-self: flex-start;
+    color: #ededf0;
+    background: #303036;
+    line-height: 1.45;
+}
+
+.assistant-message :deep(p) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)